README.md

```diff
@@ -1,2 +1,41 @@
+* The [proposals directory](./proposals) shows all the accepted proposals. See the “Implementation Status” for details on the implementation.
+* The [PRs with the `proposal` label that are closed without merging](https://github.com/mixxxdj/proposals/pulls?q=is%3Apr+label%3Aproposal+is%3Aclosed+is%3Aunmerged) show all the rejected proposals.
+
+## What’s a Design Document?
+
+It’s essential to clearly explain the reasons behind certain design decisions to have a community consensus. This is especially
+important in Mixxx, where every decision might have a significant impact given the high adoption and stability of the software and standards we work on.
+
+In our world, no decision is perfect, so having a design document explaining our trade-offs is essential.
+Such a document can also be used later as a reference and for knowledge-sharing purposes.
+
+Design documents do not always reflect what has been (or will be) implemented. Implementation details
+might have changed since a feature was merged. Design docs are not considered documentation and can not define a standard.
+Instead, it should explain the motivation, scope, decisions, and alternatives considered.
+
+## Proposal Process
+
+Don’t get scared to propose ideas! It’s amazing to innovate in the open and get feedback on ideas.
+
+The process of proposing a change via a design document is the following:
+
+1. Fork `github.com/mixxxdj/proposals`.
+2. Create a GitHub Pull Request with a design document in markdown format to the [proposals directory](./proposals). Make sure to use the [template](0000-00-00_template.md) as the guide for what sections should be present in the document. Put the creation date (the day you started preparing this design document) as the prefix and some unique name as the suffix in the file name. Once the PR is proposed, a maintainer will assign a `proposal` label.
+3. An automatic formatter is enabled in the repository. Use `make` locally to trigger the formatting of all markdown documents (requires a working Go environment). Use `make check` to check all links (will be done by the CI pipeline, too).
+4. After a sufficient amount of discussion, the Mixxx team will try to reach a consensus of accepting or rejecting the proposal. In the former case, the PR gets merged. In the latter case, the PR gets closed with meaningful reasons why the proposal was rejected.
+   1. To merge the PR, we need approval (consensus) from the maintainers of the related component(s).
+   2. Optionally: Find a sponsor among the Mixxx maintainers to get momentum on a change.
+
+Once the PR gets merged, the design document can change, but it requires (less strict, but still) a PR with review and merge by a maintainer.
+
+## Credits
+
+This proposal process was largely inspired by the [Prometheus Proposal Process](https://github.com/prometheus/proposals)
```

Comment:
I took the liberty to credit Prometheus community for the initial effort. Let me know if you think it is inappropriate or insufficient